test/functional/vimscript/system_spec.lua

```diff
@@ -567,14 +567,14 @@ describe('shell :!', function()
       4
       2]])
     if is_os('win') then
-      feed(':4verbose %!sort /R<cr>')
+      feed(':4verbose %!& sort /R<cr>')
       screen:expect {
-        any = [[Executing command: .?& { Get%-Content .* | & sort /R } 2>&1 | %%{ "$_" } | Out%-File .*; exit $LastExitCode"]],
+        any = [[Executing command: "& sort /R".*]],
```

Comment:
It's nuts that 'shelltemp' was triggering this elaborate wrapper. Eliminating 'shelltemp' may require some bug fixes or documented constraints, but will likely result in more reliable and predictable behavior.

Reply:
Yes, removing those extras should simplify things and after removing `shelltemp` make the process more strait forward and less error prone (I hope :) ), but will introduce a breaking change in some cases I guess.